Makefile

```diff
@@ -424,10 +434,6 @@ ifneq (,$(findstring "$(PLATFORM)", "linux" "gnu_kfreebsd" "kfreebsd-gnu" "gnu")
   ifeq ($(ARCH),x86)
     # linux32 make ...
     BASE_CFLAGS += -m32
-  else
-  ifeq ($(ARCH),ppc64)
-    BASE_CFLAGS += -m64
-  endif
```

Comment:
There is no need to declare `-m64` because we have specify `-mcpu=power7|power8` that automatically set `-m64`